subprojects/core-api/src/main/java/org/gradle/api/artifacts/MutableVersionConstraint.java

```diff
@@ -73,6 +73,16 @@
      */
     void prefer(String version);
 
+    /**
+     * Apply the version constraint to other occurrences of the same module in the subgraph of dependencies.
+     * This effectively gives this constraint a higher weight and allows for the accetance of previously
```

Comment:
Not sure about "allows for" vs "allows", see https://english.stackexchange.com/a/30088

:x: `accetance` -> `acceptance`